File: sdks/csharp/src/NovaBilling/Webhooks/Requests/WebhooksControllerFlutterwaveRequest.cs
using System.Text.Json.Serialization;
using NovaBilling.Core;

namespace NovaBilling;

[Serializable]
public record WebhooksControllerFlutterwaveRequest
{
    /// <summary>
    /// Flutterwave verification hash
    /// </summary>
    [JsonIgnore]
    public string? VerifHash { get; set; }

    /// <inheritdoc />
    public override string ToString()
    {
        return JsonUtils.Serialize(this);
    }
}

File: sdks/php/src/Webhooks/Requests/WebhooksControllerFlutterwaveRequest.php
<?php

namespace NovaBilling\Webhooks\Requests;

use NovaBilling\Core\Json\JsonSerializableType;

class WebhooksControllerFlutterwaveRequest extends JsonSerializableType
{
    /**
     * @var ?string $verifHash Flutterwave verification hash
     */
    public ?string $verifHash;

    /**
     * @param array{
     *   verifHash?: ?string,
     * } $values
     */
    public function __construct(
        array $values = [],
    ) {
        $this->verifHash = $values['verifHash'] ?? null;
    }
}
